src/components/collab/UserRequestHandler.ts

```diff
@@ -0,0 +1,118 @@
+import amqp from "amqplib/callback_api";
```

Comment:
Idt this belongs here? Prob in utils or something? Actually idt this code is used in the front end isit? I think this belongs in server

Reply:
Yeah I'll put it in the server file since no other file is using it